cmake/beman.inplace_vector-config.cmake.in

```diff
@@ -0,0 +1,4 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
+check_required_components("@PROJECT_NAME@")
```

Comment:
I question the value of using all the machinery of `configure_package_config_file()` just so we can call `check_required_components()`.

I know that's what exemplar does, but I think maybe we change Exemplar. This is also not a blocker, just thoughts.